tests/e2e/verfication_post_day1_test.go

```diff
@@ -61,22 +78,34 @@ var _ = Describe("TF Test", func() {
 				clusterAvailZones := getResp.Body().Nodes().AvailabilityZones()
 				Expect(err).ToNot(HaveOccurred())
 				Expect(getResp.Body().MultiAZ()).To(Equal(profile.MultiAZ))
-				if clusterAvailZones != nil {
+
+				if profile.Zones != "" {
 					Expect(clusterAvailZones).
 						To(Equal(H.JoinStringWithArray(profile.Region, zonesArray)))
-				} else {
-					Expect(clusterAvailZones).To(Equal(nil))
 				}
 			})
 		})
 		Context("Author:smiron-High-OCP-68423 @OCP-68423 @smiron", func() {
 			It("Verify compute_labels are set post cluster creation", CI.Day1Post, CI.High, func() {
 				getResp, err := CMS.RetrieveClusterDetail(CI.RHCSConnection, clusterID)
 				Expect(err).ToNot(HaveOccurred())
+
 				if profile.Labeling {
 					Expect(getResp.Body().Nodes().ComputeLabels()).To(Equal(CON.DefaultMPLabels))
 				} else {
-					Expect(getResp.Body().Nodes().ComputeLabels()).To(Equal(""))
+					Expect(getResp.Body().Nodes().ComputeLabels()).To(Equal(CON.NilMap))
```

Comment:
```suggestion
					Expect(getResp.Body().Nodes().ComputeLabels()).To(BeNil())
```

Reply:
I changed to `BeNil` and reciving this err:
```
  [FAILED] Expected
      <map[string]string | len:0>: nil
  to equal
      <*matchers.BeNilMatcher | 0x1044caf40>: {}
```
it seems like even the nil of the empty map, is of type `string`